app/src/flavorAtest/java/net/gsantner/opoc/util/GsAndroidSupportMeWrapper.java

```diff
@@ -56,11 +56,11 @@ public void mainOnResume(final Context context) {
         }
```

Comment:
Studio thinks this entire class / file is unused. Not sure why, but if so, do we want to keep it?

Reply:
It is currently unused yes. But you can just leave it